pkg/apis/internal.admin.acorn.io/v1/volumes.go

```diff
@@ -0,0 +1,46 @@
+package v1
+
+import (
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ProjectVolumeClassInstance struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	StorageClassName   string         `json:"storageClassName"`
+	Description        string         `json:"description"`
+	Default            bool           `json:"default,omitempty"`
+	AllowedAccessModes v1.AccessModes `json:"allowedAccessModes,omitempty"`
+	DefaultSize        v1.Quantity    `json:"defaultSize,omitempty"`
+	MinSize            v1.Quantity    `json:"minSize,omitempty"`
+	MaxSize            v1.Quantity    `json:"maxSize,omitempty"`
+	Inactive           bool           `json:"inactive,omitempty"`
+}
```

Comment:
Nit-pick with a pretty large amount of changes implied so I won't block for it. Thoughts, though?

```suggestion
type ProjectVolumeClassInstance struct {
	metav1.TypeMeta   `json:",inline"`
	metav1.ObjectMeta `json:"metadata,omitempty"`

	StorageClassName   string         `json:"storageClassName"`
	Description        string         `json:"description"`
	Default            bool           `json:"default,omitempty"`
	AllowedAccessModes v1.AccessModes `json:"allowedAccessModes,omitempty"`
	Size               Size           `json:"size,omitempty"`
	Inactive           bool           `json:"inactive,omitempty"`
}

type VolumeSize struct {
	Default  v1.Quantity    `json:"default,omitempty"`
	Min      v1.Quantity    `json:"min,omitempty"`
	Max      v1.Quantity    `json:"max,omitempty"`
}
```

Reply:
Done.